azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/ConfigMojo.java

```diff
@@ -136,7 +146,7 @@ protected boolean confirmConfiguration(WebAppConfiguration configuration) throws
             System.out.println("Slot name : " + slotSetting.getName());
             System.out.println("ConfigurationSource : " + slotSetting.getConfigurationSource());
         }
-        final String result = queryer.assureInputFromUser("confirm", "Y", BOOLEAN_REGEX, "Confirm (Y/N)? : ", null);
+        final String result = queryer.assureInputFromUser("confirm", "Y", BOOLEAN_REGEX, "Confirm (Y/N)", null);
```

Comment:
how about replacing the values of string (such as "Y", "N", etc) with constant variables?